### PR TITLE
appveyor: Shard more slow MSVC builders

### PR DESCRIPTION
**Pull request by alexcrichton**
Take two more slow builders and split them in two to get them hopefully to around 2 hrs

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -19,12 +19,18 @@ environment:
...
+    RUST_CONFIGURE_ARGS: --build=i686-pc-windows-msvc
+    SCRIPT: python x.py test src/test/run-pass src/test/compile-fail
 
   # MSVC aux tests
   - MSYS_BITS: 64
-    RUST_CHECK_TARGET: check-aux
+    RUST_CHECK_TARGET: check-aux AUX_ARGS="--exclude src/tools/cargotest --exclude src/tools/cargo"
+    RUST_CONFIGURE_ARGS: --build=x86_64-pc-windows-msvc
+  - MSYS_BITS: 64
+    SCRIPT: python x.py test src/test/run-pass src/tools/cargotest src/tools/cargo
```
Comment (Mark-Simulacrum) [inline appveyor.yml]: Hm, this should probably be up above? The ordering here looks a little odd -- maybe I'm not entirely following though.

**Comment by Mark-Simulacrum** [comment]
r=me pending maybe changing the order to be exclude/include instead of the somewhat odd order that I see right now.

File: appveyor.yml (hunk 1 above)
Comment (kennytm) [reply appveyor.yml]: If we are including `run-pass` here, the `x86_64-pc-windows-msvc` job (the 1st job) should have `--exclude run-pass`.

Comment (alexcrichton) [reply appveyor.yml]: Oh oops the `run-pass` was a mistake here, I'll remove that.

**Comment by kennytm** [comment]
Oh I thought the run-pass move was intended, since there were three 3-hour-timeout due to `check x86_64-pc-windows-msvc` last week too.

**Comment by alexcrichton** [comment]
Oh ok, we don't have enough builders to split that as well. Let's try this sharding and then see if we need another?

**Comment by kennytm** [comment]
I mean one of the two new check-aux job will likely be finished in less than 2 hours (I guess the original one since it only runs pretty tests), and perhaps we could move the run-pass test into that job, without increasing the number of builders.

**Comment by alexcrichton** [comment]
Oh yeah for sure, I figured we'd see the timings first and evaluate after that.

**Comment by alexcrichton** [comment]
@bors: retry

prioritizing rollup with this in it

**Comment by alexcrichton** [comment]
@bors: retry

prioritizing rollup with this in it

**Comment by alexcrichton** [comment]
@bors: r-


this [will fail](https://ci.appveyor.com/project/rust-lang/rust/build/1.0.6595/job/et71cgb0sxh82auh), fill [fix this in the rollup](https://github.com/rust-lang/rust/pull/48806)

**Pull request merged**